assets/js/app.js

```diff
@@ -0,0 +1,12 @@
+const hamburger = document.querySelector('.hamburger');
+const navMenu = document.querySelector('.nav-menu');
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+
+document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  navMenu.classList.remove('active');
+}));
```

Comment:
While you did an impressive job working on the project requirements i notice a few errors that need to be corrected It would be nice to carefully inspect the figma template you picked. Plese set your font size to 32px and your line height to 44px to match the design of the template design on figma. I also notice some of your page content is visible when the mobile menu is open please the page content should not be visible in the mobile menu when it open its should all be white and the only things that need to be seen are the navigation links.

This is your design :
![shot 1](https://user-images.githubusercontent.com/79506105/216932947-c05043e3-853a-41c4-912b-ea544df13d7b.png)

Please follow this figma design:

![Screenshot 2023-02-06 at 11 07 07 AM](https://user-images.githubusercontent.com/79506105/216933197-026daba1-9326-45ae-a911-314ea92a2755.png)